db/schema.rb

```diff
@@ -125,6 +125,13 @@
     t.datetime "updated_at",                          null: false
   end
 
+  create_table "products", force: :cascade do |t|
```

Comment:
You regenerated the schema.rb from an old version of the db.  The latest schema.rb from development doesn't have a Product table.  Also you removed Vendor when you regenerated.  Make sure you run a rake db:migrate with the latest migrations to get a properly generated file and then commit this fix. !fix  !fix  !resolve  !fix